rasa/shared/nlu/training_data/features.py

```diff
@@ -136,3 +158,224 @@ def fingerprint(self) -> Text:
         return rasa.shared.utils.io.deep_container_fingerprint(
             [self.type, self.origin, self.attribute, f_as_text]
         )
+
+    @staticmethod
+    def filter(
```

Comment:
But this is `Features.filter` and not the builtin `filter` - @wochinge  ?

Reply:
Yeah, doesn't collide and in my opinion clear as it's called on  the class / object. If it would be a function (not attached to a class) then I'd be with Johannes 👍🏻